tv_apps/smart-system/js/app_modal_dialog.js

```diff
@@ -345,7 +398,8 @@
   };
 
   AppModalDialog.prototype.hide = function amd_hide() {
-    this.element.blur();
+    document.activeElement.blur();
```

Comment:
May we call blur on the focused element that we know? I feel the focus issue you met may be triggered by this kind of code. We should only call blur on the focused element of simpleKeyNavigation.